libraries/tokens/CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.0] - 2020-11-26
+
+### Added
+
+- Typescript support in all of the EDS libraries ([Typescript Milestone](https://github.com/equinor/design-system/milestone/7?closed=1))
```

Comment:
```suggestion
- Types, as part of the ([Typescript Milestone](https://github.com/equinor/design-system/milestone/7?closed=1))
```

Since Typescript _support_ is something an editor has for example, while the _types_ are what we provide. Or maybe _Typescript types_ if we want to spell it out.